src/components/Grid/Grid.css

```diff
@@ -0,0 +1,270 @@
+  &--fullWidth {
+    max-width: 100%;
+  }
+
+  /* nested grid containers */
+  & .grid-container {
+    @inherit .mx-0, .px-0;
+  }
+}
+
+/* 
+ * Rows .row-N - unlimited number
+ * 
+ * Rows have an implicit height but you can use the row class to 
+ * change the presentation order of cells 
+   ========================================================================== */
+@variants responsive {
+  @for $i from 1 to 100 {
+    .row-$i {
+      grid-row: $(i);
+    }
+  }
+}
+
+[class*='row-'] {
+  /* nested grid containers */
+  & > .grid-container {
+    margin: 0;
+    padding: 0;
+  }
+}
+
+/* 
+ * Cells (columns) .cell-N - 12 in total
+ * 
+ * Use the .cell-N class to set the width of individual cells.
+ * Multi-cell elements span across gutters, so .cell-2 spans the first cell
+ * first gutter and second cell stopping at before second gutter
+ *
+ * RESPONSIVE: .sm:cell-N .md:cell-N .lg:cell-N 
+ * corresponding css breakpoint classes (.sm:, .md:, .lg:) are available
+ * to change the cell width at specified breakpoints giving you tons of flexibility
+   ========================================================================== */
+   @variants responsive {
+      @for $i from 1 to 12 {
+        .cell-$i {
+          grid-column: span $(i);
+          -ms-grid-column-span: $(i);
+          /* 
+           * An initial setting on grid items is min-width: auto. 
+           * This means that an item cannot be shorter than its content. 
+           * You can override this default with min-width: 0 
+           */
+          min-width: 0;
+        }
+      }
+  }
+
+[class*='cell-'] {
+  /* nested grid containers */
+  & > .grid-container {
+    margin: 0;
+    padding: 0;
+  }
+}
+
+/* 
+ * Cell offset/start .cell-start-N - 12 in total
+ * 
+ * This specifies what column a cell should start at. 
+ * For example you can start a cell at column 6 (.cell-start-N) and have 
+ * it span 6 columns (.cell-6) making it go from the middle to the right 
+ * edge of the grid. 
+ *
+ * SHORTHAND: .cell-<start>-<span>
+ * ex. .cell-3-6 = start at cell 3 and span 6 cells 
+ *
+ * RESPONSIVE: .sm:cell-N-N .sm:cell-start-N
+ * corresponding css breakpoint classes (.sm:, .md:, .lg:) are available
+ * to change the cell width at specified breakpoints giving you tons of flexibilit
+   ========================================================================== */
+   @variants responsive {
+
+    @for $i from 1 to 12 {
+      @for $x from 11 to 1 {
+        .cell-$i.cell-start-$x {
+          grid-column: $(x) / span $(i);
+    
+          min-width: 0;
+        }
+        /* shorthand cell-<start>-<span> */
+        .cell-$(x)-$(i) {
+          grid-column: $(x) / span $(i);
+          min-width: 0;
+        }
+      }
+    }
+    
+  }
+
+/* Tablet
+  ========================================================================== */
+
+
+
+/* =======================================================================
+  IE 11 css grid fallback
```

Comment:
Maybe all the IE stuff could live in another file for organizational reasons?

Reply:
tried that but couldn't get the `@import` order in tailwind.src.css to work properly since the media query has to come after the rest of it